### ISSUE TITLE
RustDesk crash my pc (Ubuntu 22.04)

### ISSUE THREAD

**Issue by XBlayz** [rustdesk/doc.rustdesk.com#38]
I have installed the app using `sudo apt install -fy ./rustdesk-1.1.9.deb` and I have switched to X11 using the "Fix it" button, I have reboot and now when I try to launch the app freeze my monitor and completely crash my laptop (I have to force shutdown).

**Comment by rustdesk** [comment]
Really sorry for this. We did tested on 22.04 before. We are removing the stupid "Fix it" in new version.

**Comment by rustdesk** [comment]
Wondering if this https://linuxconfig.org/how-to-enable-disable-wayland-on-ubuntu-22-04-desktop can help you.

**Comment by XBlayz** [comment]
Now it didn't crash but when I try to connect it say:
`Unsupported display server type wayland, x11 expected`

**Comment by rustdesk** [comment]
Yes, it does not support Wayland yet, that's why you saw "Fix it". We are stil working on Wayland support.

**Comment by rustdesk** [comment]
Follow us on twitter https://twitter.com/rustdesk for update.
